test/notBoth.js

```diff
@@ -36,15 +49,17 @@ describe('notBoth', function() {
     eq(z.notCalled, true);
   });
 
-  it('accepts fantasy-land applicative functors', function() {
-    eq(RA.notBoth(Just(true), Just(true)), Just(false));
-    eq(RA.notBoth(Just(true), Just(false)), Just(true));
-    eq(RA.notBoth(Just(false), Just(true)), Just(true));
-    eq(RA.notBoth(Just(false), Just(false)), Just(true));
-    eq(RA.notBoth(Just(true), Nothing()), Nothing());
-    eq(RA.notBoth(Nothing(), Just(true)), Nothing());
-    eq(RA.notBoth(Nothing(), Just(false)), Nothing());
-    eq(RA.notBoth(Just(false), Nothing()), Nothing());
-    eq(RA.notBoth(Nothing(), Nothing()), Nothing());
-  });
+  if (isFantasyLandSupported) {
```

Comment:
Otherwise your solution is pristine. Good job ! ;]

Reply:
I'm unsure of the change you're asking for here.

Reply:
No change required, just a general comment about your solution.

Reply:
Ok. In that case - thank you.